src/quo2/components/profile/collectible/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.profile.collectible.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.profile.collectible.style :as style]
+            [react-native.core :as rn]))
+
+(defn plus-n-tile
+  [n]
+  [rn/view {:style (merge style/bottom-right (style/plus-n))}
+   [text/text
+    {:style  (style/plus-n-text)
+     :size   :paragraph-2
+     :weight :medium}
+    (str "+" n)]])
+
+(defn tile
+  [{:keys [uri size]}]
+  [rn/image
+   {:style  (style/tile-style-by-size size)
+    :source {:uri uri}}])
+
+(defn two-tiles
+  [{:keys [imgs size]}]
+  [:<>
+   [rn/view {:style style/top-left}
+    [tile {:uri (first imgs) :size size}]]
+   [rn/view {:style style/bottom-right}
+    [tile {:uri (second imgs) :size size}]]])
+
+(defn three-tiles
+  [{:keys [tiles size]}]
+  (let [[img-1 img-2 img-3 & _] tiles]
+    [:<>
+     [rn/view {:style style/top-left}
+      [tile {:uri img-1 :size size}]]
+     [rn/view {:style style/top-right}
+      [tile {:uri img-2 :size size}]]
+     [rn/view {:style style/bottom-left}
+      [tile {:uri img-3 :size size}]]]))
+
+(defn tile-container
+  [{:keys [imgs]}]
```

Comment:
I would prefer if we didn't shorten certain names, because in this PR they affect readability for no actual gain.

- `imgs` => `images`
- `rem-imgs` => `remaining-images`
- `first-3-imgs` => `first-3-images`

Reply:
agree

Reply:
Completely agree!

The change is in b1f93d96cc745575eea7114b68b271a28690ba83